backend-api/src/functions/tests/pact/asyncToken/asyncTokenConfiguration.ts

```diff
@@ -0,0 +1,54 @@
+import { registeredLogs } from "../../../asyncToken/registeredLogs";
+import { MockEventWriterSuccess } from "../../../services/events/tests/mocks";
+import { MockLoggingAdapter } from "../../../services/logging/tests/mockLogger";
+import { Logger } from "../../../services/logging/logger";
+import {
+  MockClientRegistryServiceBadRequestResult,
+  MockClientRegistryServiceSuccessResult,
+  MockTokenServiceSuccessResult,
+} from "../../../testUtils/asyncTokenMocks";
+import { IAsyncTokenRequestDependencies } from "../../../asyncToken/handlerDependencies";
+
+export class AsyncTokenConfiguration {
```

Comment:
Please may we align this with our solution for the credential handler?

Reply:
I believe the solutions are currently the same. Is there anything in particular you want updated?

Reply:
Updated